ydb/public/api/protos/ydb_persqueue_v1.proto

```diff
@@ -1079,6 +1079,43 @@ message Credentials {
     }
 }
 
+enum AutoScalingStrategy {
+    // Auto scaling disabled
+    DISABLED = 0;
+    // The auto scaling may only increase the number of partitions
+    SCALE_UP = 1;
+    // The auto scaling may increase and decrease the number of partitions
+    SCALE_UP_AND_DOWN = 2;
```

Comment:
Just to be considered: though this enum itself is deprecated, it's values looks more natural and understandable for me: https://github.com/ydb-platform/ydb/blob/main/ydb/public/api/protos/ydb_table.proto#L288